.buildkite/steps/test-demo-notebooks.sh

```diff
@@ -39,8 +39,9 @@ echo "--- :python: installing papermill"
 # Pulling in https://github.com/nteract/papermill/pull/459 for --execution-timeout, which hasn't been released yet
 pip install https://github.com/nteract/papermill/archive/master.tar.gz
 
-echo "--- installing dependencies"
-pip install -q --no-cache-dir '.[demos]'
+echo "--- :python: installing stellargraph"
+# install stellargraph itself, which (hopefully) won't install any dependencies
+pip install .
```

Comment:
I was curious why `pip install .` is in this script, and not in `.buildkite/steps/script.sh` unless I missed it?

Reply:
From slack (as referenced in your approval comment):

> I think: the notebooks run from a subdirectory, so `import stellargraph` in them cannot pick up the `stellargraph/__init__.py`, but tests run at the top level of the repo and so can.